docs/configuration.md

```diff
@@ -200,10 +200,10 @@ Default: `['clear-text', 'progress', 'html']`<br />
 Command line: `--reporters clear-text,progress,dots,dashboard,html,json`<br />
 Config file: `"reporters": ["clear-text", "progress", "dots", "dashboard", "html", "json"]`
 
-With `reporters`, you can set the reporters for stryker to use.
+With `reporters`, you can set the reporters for Stryker to use.
 These reporters can be used out of the box: `html`, `json`, `progress`, `clear-text`, `dots`, `dashboard` and `event-recorder`.
 By default, `clear-text`, `progress`, `html` are active if no reporters are configured. See [reporter plugins](./plugins.md#reporters)
-for a Fdescription on each reporter.
+for a fuller description of each reporter.
```

Comment:
I guessed at what this was meant to be; reads okay to me now.

Reply:
maybe just "full" description? No need for "fuller" IMO :D

Reply:
Done.